%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% ELIFE ARTICLE TEMPLATE
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% PREAMBLE
\documentclass[9pt,lineno]{elife}
% Use the onehalfspacing option for 1.5 line spacing
% Use the doublespacing option for 2.0 line spacing
% Please note that these options may affect formatting.
% Additionally, the use of the \newcommand function should be limited.

\usepackage{lipsum} % Required to insert dummy text
\usepackage[version=4]{mhchem}
\usepackage{siunitx}
\DeclareSIUnit\Molar{M}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% ARTICLE SETUP
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\title{Effects of delayed strain submission and vaccine development on long-term forecast accuracy of seasonal influenza A/H3N2}

\author[1*]{John Huddleston}
\author[2]{Trevor Bedford}
\affil[1]{Vaccine and Infectious Disease Division, Fred Hutchinson Cancer Center, Seattle, WA, USA}
\affil[2]{Howard Hughes Medical Institute, Seattle, WA, USA}

\corr{jhuddles@fredhutch.org}{JH}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% ARTICLE START
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{document}

\maketitle

\begin{abstract}
TKTK
\end{abstract}

\section{Introduction}

TKTK

\begin{figure}[htb]
\includegraphics[width=\linewidth]{figures/distribution_of_delays_and_horizons}
\caption{Model of submission delays and forecast horizons.}
\label{fig:model_of_delays_and_horizons}
\end{figure}

\section{Results}

\subsection{Reducing forecast horizons decreases distances between predicted and observed future populations}

Previously, we trained long-term forecasting models that minimized the earth mover's distance \citep{Rubner1998} between predicted and observed future populations \citep{Huddleston2020}.
Each forecast predicted 12 months into the future from a given initial timepoint to a final timepoint.
Specifically, the forecasts predicted the future frequencies of samples circulating at the initial timepoint based on the estimated fitness of each sample.
We calculated the distance between hemagglutinin (HA) amino acid sequences observed at the initial timepoint and the final timepoint, weighting the distance by the predicted future frequency of samples at the initial timepoint and the observed frequency of samples at the final timepoint.
The best forecasting models used fitness estimates that reduced the distance between predicted and observed future populations.
For example, the most accurate sequence-only model for the 12-month forecast horizon estimated fitness with local branching index (LBI) \citep{Neher:2014eu} and mutational load \citep{Luksza:2014hj}.

Here, we produced forecasts 3, 6, 9, and 12 months into the future using HA sequences available at each initial timepoint based on one of three scenarios of delayed sequence submission to public databases: no delay, an ``ideal'' delay ($\sim$1-month average), and a ``realistic'' delay ($\sim$3-month average) (\FIG{model_of_delays_and_horizons}).
For natural A/H3N2 populations, we used the best sequence-only model, LBI and mutational load, which we previously trained on 12-month forecasts without any submission delay.
For simulated A/H3N2-like populations, we used the observed fitness per sample provided by the simulator.
For each forecast horizon and submission delay type, we calculated the earth mover's distance between the predicted future populations under the given delay scenario and the observed future populations without any delay in sequence availability.
We anticipated that reducing either the forecast horizon or the submission delay would reduce the distance to the future in amino acids (AAs), representing increased accuracy of the forecasting models.

As expected, we found that reducing the forecast horizon from the current standard of 12 months linearly reduced the distance to the future population predicted by the LBI and mutational load model (\FIG{h3n2_distances_to_the_future}).
Under the all three submission delay scenarios, the distance to the future reduced by approximately 1 AA on average for each 3-month reduction in forecast horizon (\TABLE{h3n2_distances_to_the_future}).
We observed the greatest average reduction in distance to the future ($\sim$1.4 AAs) between the 6- and 3-month forecast horizons.
Reducing the forecast horizon also noticeably reduced the variance per timepoint in predicted future populations across all delay scenarios (\FIG{h3n2_distances_to_the_future}).
For example, the standard deviation of distances to the future reduced from $\sim$2.6 AAs at the 12-month horizon to $\sim$1 AA at the 3-month horizon (\TABLE{h3n2_distances_to_the_future}).
We observed the same pattern for simulated A/H3N2-like populations (\FIGSUPP[h3n2_distances_to_the_future]{simulated_distances_to_the_future}).
Thus, reducing how far we have to predict into the future increased both forecast accuracy and precision.

In contrast, we found that reducing submission delays from a $\sim$3-month average delay in the realistic scenario to a $\sim$1-month average delay in the ideal scenario had relatively little effect on distance to the future.
At the 12-month forecast horizon, the ideal and realistic delay scenarios produced similar predictions, with the only noticeable improvement observed under the scenario without any submission delays (\FIG{h3n2_distances_to_the_future}).
As the forecast horizon decreased, the effect of submission delays appeared more prominent, with the greatest effect of reduced delays observed at the 3-month forecast horizon.
However, the average improvement from the realistic to the ideal submission delay scenario at the 3-month horizon was still only $~\sim$0.3 AAs (\TABLE{h3n2_distances_to_the_future}).
Reducing submission delays also had little effect on the variance per timepoint in predicted future populations.
Interestingly, we observed a stronger effect of reducing submission delays in simulated A/H3N2-like populations (\FIGSUPP[h3n2_distances_to_the_future]{simulated_distances_to_the_future}).
We found the best average improvement between realistic and ideal delays of $\sim$0.7 AAs at the 3-month horizon.
As with natural A/H3N2 populations, the effect of reducing submission delays appeared to increase as the forecast horizon decreased.
These results indicate that reducing submission delays may have little effect under the current 12-month forecast approach used for influenza vaccine composition, but reducing submission delays could be increasingly important as we forecast from closer to future influenza populations.

\begin{figure}[htb]
\includegraphics[width=\linewidth]{figures/h3n2_distances_to_the_future_by_delay_and_horizon}
\caption{Distances to the future per timepoint (AAs) for natural A/H3N2 populations by forecast horizon and submission delay type based on forecasts from the local branching index (LBI) and mutational load model.
  Each point represents a final timepoint whose population was predicted from the number of months earlier corresponding to the forecast horizon.
  Points are colored by submission delay type including forecasts made with no delay (blue), an ideal delay (orange), and a realistic delay (green).}
\label{fig:h3n2_distances_to_the_future}
%
\figsupp[Distances to the future for simulated H3N2-like populations]
{Distances to the future per timepoint (AAs) for simulated H3N2-like populations by forecast horizon and submission delay type.
  \figsuppdata{Distances to the future for simulated H3N2-like populations; see \url{https://doi.org/xxx}}}
{\includegraphics[width=6cm]{figures/simulated_distances_to_the_future_by_delay_and_horizon}}\label{figsupp:simulated_distances_to_the_future}
%
\figdata{Distances to the future for natural H3N2 populations.}\label{figdata:h3n2_distances_to_the_future}
\figsrccode{Jupyter notebook used to produce this figure and the supplemental figure lives in \texttt{workflow/notebooks/plot-distances-to-the-future-by-delay-type-and-horizon-for-population.py.ipynb}.}\label{figsrccode:distances_to_the_future}
\end{figure}

\begin{table}[htb]
  \begin{center}
    \input{tables/h3n2_distances_to_the_future_by_delay_and_horizon.tex}
    \caption{Distances to the future in amino acids (mean +/- standard deviation AAs) by forecast horizon (in months) and submission delay for H3N2 populations.}
    \label{tab:h3n2_distances_to_the_future}
  \end{center}
\end{table}

\subsection{Current clade frequency errors}

Although the distance between predicted and observed future populations in amino acids provides an unbiased metric to optimize forecasting models, in practice, we use these models to forecast clade frequencies.
To produce these forecasts, we estimate the frequencies of extant clades at an initial timepoint and predict their future frequencies from those initial frequencies and the fitness estimates of each clade's tips.
Given the importance of initial clade frequencies in these forecasts, we tested the effect of submission delays on current clade frequency estimates.
For each timepoint in our analysis, we summed the frequency per clade assigned to each tip (see Methods).
We calculated the frequency error per timepoint and clade as the difference between the clade frequency without submission delays and the frequency with either an ideal or realistic delay.

\begin{figure}[htb]
\includegraphics[width=\linewidth]{figures/h3n2_current_frequency_errors_by_delay}
\caption{Clade frequency errors for natural A/H3N2 clades at the same timepoint calculated as the difference between clade frequencies without submission delay and corresponding frequencies with either A) ideal or B) realistic submission delays.
Distributions of frequency errors appear normally distributed in both delay scenarios for both C) small clades ($\ge$1\% and $<$10\% frequency) and D) large clades ($\ge$10\%).}
\label{fig:h3n2_current_clade_frequency_errors}
%
\figsupp[Current clade frequency errors for simulated H3N2-like populations]
{Clade frequency errors between simulated H3N2-like HA populations with ideal or realistic submission delays and populations without any submission delay.
  \figsuppdata{Current frequencies per tip in each simulated H3N2-like tree of the forecast analysis by delay type (none, ideal, and realistic); see \url{https://doi.org/xxx}}}
{\includegraphics[width=6cm]{figures/simulated_current_frequency_errors_by_delay}}\label{figsupp:simulated_current_clade_frequency_errors}
%
\figdata{Current frequencies per tip in each natural H3N2 tree of the forecast analysis by delay type (none, ideal, and observed).}\label{figdata:h3n2_tip_attributes}
\figsrccode{Jupyter notebook used to produce this figure and the supplemental figure lives in \texttt{workflow/notebooks/plot-current-clade-frequency-errors-by-delay-type-for-populations.py.ipynb}.}\label{figsrccode:current_clade_frequency_errors}
\end{figure}

Surprisingly, we found that even with an average $\sim$1-month delay in submissions, clade frequencies in the ideal delay scenario almost exactly matched frequencies without any delays at all (\FIG{h3n2_current_clade_frequency_errors}A).
For both small clades ($\ge$1\% and $<$10\% frequency without delays) and large clades ($\ge$10\% frequency), clade frequency errors for ideal delays rarely exceeded 1\% (\FIG{h3n2_current_clade_frequency_errors}C and D).
In contrast, realistic delays produced higher errors (\FIG{h3n2_current_clade_frequency_errors}B), ranging between -3\% and 3\% for small clades (\FIG{h3n2_current_clade_frequency_errors}C) and -5\% and 6\% for large clades (\FIG{h3n2_current_clade_frequency_errors}D).
For both delay scenarios, errors appeared normally distributed without a bias toward over- or underestimation of current clade frequencies.
These results indicate that reducing submission delays for influenza sequences from a 3-month to 1-month average could effectively eliminate most of our error in current clade frequency estimates.

Delayed submissions more strongly affected clade frequencies for simulated A/H3N2-like populations (\FIGSUPP[h3n2_current_clade_frequency_errors]{simulated_current_clade_frequency_errors}).
With ideal delays, small clade errors ranged from -5\% to 6\% and large clade errors from -7\% to 13\% (\FIGSUPP[h3n2_current_clade_frequency_errors]{simulated_current_clade_frequency_errors}C and D).
As expected, realistic delays produced higher errors ranging from -13\% to 8\% for small clades and -16\% to 35\% for large clades.
Delays also induced a bias toward underestimation of large clade frequencies with a mean and median error of 1\% with ideal delays and 3\% with realistic delays (\FIGSUPP[h3n2_current_clade_frequency_errors]{simulated_current_clade_frequency_errors}D).
This increased error in frequency estimation for simulated populations corresponds with an overall faster evolutionary rate compared to natural A/H3N2 populations, as we observed in the greater distances to the future for simulated populations (\FIGSUPP[h3n2_distances_to_the_future]{simulated_distances_to_the_future}).
Correspondingly, we would expect seasonal influenza lineages that evolve slower than A/H3N2 (A/H1N1pdm and B/Victoria) to have lower errors in current clade frequency estimates as a result of submission delays.

\subsection{Forecast clade frequency errors}

\begin{figure}[htb]
\includegraphics[width=\linewidth]{figures/h3n2_forecast_frequency_errors_by_delay_and_horizon}
\caption{Forecast clade frequency errors for natural H3N2 populations by forecast horizon in months and submission delay type (none, ideal, or observed).}
\label{fig:h3n2_forecast_clade_frequency_errors}
%
\figsupp[Absolute forecast clade frequency errors for H3N2 populations.]
{Absolute forecast clade frequency errors for natural H3N2 populations by forecast horizon in months and submission delay type (none, ideal, or observed).
  \figsuppdata{Current, estimated future, and observed future clade frequencies per initial timepoint, forecast horizon, and submission delay type for H3N2 populations; see \url{https://doi.org/xxx}}}
{\includegraphics[width=6cm]{figures/h3n2_absolute_forecast_frequency_errors_by_delay_and_horizon}}\label{figsupp:h3n2_absolute_forecast_clade_frequency_errors}
%
\figsupp[Forecast clade frequency errors for simulated H3N2-like populations.]
{Forecast clade frequency errors for simulated H3N2-like HA populations by forecast horizon in months and submission delay type (none, ideal, or realistic).
  \figsuppdata{Current, estimated future, and observed future clade frequencies per initial timepoint, forecast horizon, and submission delay type for simulated H3N2-like populations; see \url{https://doi.org/xxx}}}
{\includegraphics[width=6cm]{figures/simulated_forecast_frequency_errors_by_delay_and_horizon}}\label{figsupp:simulated_forecast_clade_frequency_errors}
%
\figsupp[Absolute forecast clade frequency errors for simulated H3N2-like populations.]
{Absolute forecast clade frequency errors for simulated H3N2-like HA populations by forecast horizon in months and submission delay type (none, ideal, or realistic).
  \figsuppdata{Current, estimated future, and observed future clade frequencies per initial timepoint, forecast horizon, and submission delay type for simulated H3N2-like HA populations; see \url{https://doi.org/xxx}}}
{\includegraphics[width=6cm]{figures/simulated_absolute_forecast_frequency_errors_by_delay_and_horizon}}\label{figsupp:simulated_absolute_forecast_clade_frequency_errors}
%
\figdata{Current, estimated future, and observed future clade frequencies per initial timepoint, forecast horizon, and submission delay type for H3N2 populations.}\label{figdata:h3n2_clade_frequencies}
\figsrccode{Jupyter notebook used to produce this figure and the supplemental figures lives in \texttt{workflow/notebooks/plot-forecast-clade-frequency-errors-by-delay-type-and-horizon-for-population.py.ipynb}.}\label{figsrccode:forecast_clade_frequency_errors}
\end{figure}

\begin{table}[htb]
  \begin{center}
    \input{tables/h3n2_forecast_frequency_errors_future_by_delay_and_horizon.tex}
    \caption{Errors in clade frequencies between observed and predicted values by forecast horizon (in months) and submission delay for H3N2 clades with an initial frequency $\geq$10\%.}
    \label{tab:h3n2_forecast_frequency_errors}
  \end{center}
\end{table}

\begin{table}[htb]
  \begin{center}
    \input{tables/h3n2_absolute_forecast_frequency_errors_future_by_delay_and_horizon.tex}
    \caption{Absolute errors in clade frequencies between observed and predicted values by forecast horizon (in months) and submission delay for H3N2 clades with an initial frequency $\geq$10\%.}
    \label{tab:h3n2_absolute_forecast_frequency_errors}
  \end{center}
\end{table}

\subsection{Effects of realistic interventions on forecast accuracy}

\begin{figure}[htb]
\includegraphics[width=\linewidth]{figures/h3n2_effects_of_realistic_interventions}
\caption{Improvement of clade frequency errors for H3N2 populations between the status quo and realistic interventions.}
\label{fig:h3n2_effects_of_realistic_interventions}
%
\figsupp[Improvement of clade frequency errors for simulated H3N2-like populations]
{Improvement of clade frequency errors for simulated H3N2-like populations between the status quo and realistic interventions.
  \figsuppdata{Differences in absolute clade frequency error per future timepoint and clade between the status quo and realistic interventions for simulated H3N2-like populations; see \url{https://doi.org/xxx}}}
{\includegraphics[width=6cm]{figures/simulated_effects_of_realistic_interventions}}\label{figsupp:simulated_effects_of_realistic_interventions}
%
\figdata{Differences in absolute clade frequency error per future timepoint and clade between the status quo and realistic interventions for H3N2 populations.}\label{figdata:h3n2_effects_of_realistic_interventions}
\figsrccode{Jupyter notebook used to produce this figure and the supplemental figure lives in \texttt{workflow/notebooks/plot-forecast-clade-frequency-errors-by-delay-type-and-horizon-for-population.py.ipynb}.}\label{figsrccode:effects_of_realistic_interventions}
\end{figure}

\begin{table}[htb]
  \begin{center}
    \input{tables/h3n2_effects_of_realistic_interventions.tex}
    \caption{Improvement in absolute clade frequency errors between the status quo (12-month forecast horizon and realistic submission delay) and realistic interventions for H3N2 clades with an initial frequency of $\geq$10\%.}
    \label{tab:h3n2_effects_of_realistic_interventions}
  \end{center}
\end{table}

\section{Discussion}

TKTK

\section{Methods and Materials}

\subsection{Estimating and assigning submission delays}

We estimated the delay between sample collection and submission of A/H3N2 hemagglutinin (HA) sequences to the GISAID database \citep{gisaid} by calculating the difference in GISAID-annotated submission date and collection date in days for samples collected between January 1, 2019 and January 1, 2020 and with a submission date prior to October 1, 2020.
We selected this period of time as representative of modern genomic surveillance efforts prior to changes in circulation patterns of influenza caused by the SARS-CoV-2 pandemic.
Of the 104,392 HA sequences in GISAID, 11,222 (11\%) were collected during this period with a mean submission delay of 98 days ($\sim$3 months) and a median delay of 74 days.
Only 11\% of sequences (N=1,210) were submitted within 4 weeks of collection, and only 36\% (N=4,057) were submitted within 8 weeks (\FIG{model_of_delays_and_horizons}A, purple).

We modeled the shape of the observed delay distribution as a gamma distribution using a maximum likelihood fit from SciPy 1.10.1 \citep{scipy}.
With this approach, we estimated a shape parameter of 1.76, a scale parameter of 53.18, and location parameter of 3.98.
The product of these shape and scale values corresponded to a mean delay of 93.76 days (\FIG{model_of_delays_and_horizons}A, green).
To assign realistic submission delays to each sample in our analysis, we randomly sampled from this gamma distribution and calculated a ``realistic submission date'' by adding the sampled delay in days to the observed collection date.

Based on the observed rapid submission of SARS-CoV-2 genomes during the first years of the pandemic, we expected that an achievable ``ideal'' submission delay for seasonal influenza sequences would have a 1-month average delay instead of the observed $\sim$3-month delay from the pre-pandemic period.
We modeled this ideal submission delay distribution by dividing the gamma shape parameter by 3 to get a value of 0.59 and a corresponding mean delay of 31.25 days (\FIG{model_of_delays_and_horizons}A, orange).
This approach effectively shifted the realistic gamma toward zero, while maintaining the relatively longer upper tail of the distribution.
To assign ideal submission delays to each sample in our analysis, we randomly sampled from this modified gamma distribution and added the sampled delay in days to the observed collection date.
Additionally, we required that each sample's ``ideal'' delay be less than or equal to its ``realistic'' delay.

\subsection{Forecasting with different forecast horizons}

We tested the effect of forecasting future influenza populations at forecast horizons of 3, 6, 9, and 12 months (\FIG{model_of_delays_and_horizons}B).
Previously, we produced forecasts every 6 months starting from October 1 and April 1 and predicting 12 months into the future \citep{Huddleston2020}.
To support forecasts in 3-month intervals, we produced annotated time trees for 6 years of HA sequences every 3 months with data available up to the first day of January, April, July, and October.
We produced these trees for each timepoint with three different delay scenarios: no delay, ideal delay, and realistic delay.
For each scenario, we selected sequences for analysis at a given timepoint based on their collection date, ideal submission date, or realistic submission date, respectively.
This experimental design produced forecasts for three delay types at each of the four forecast horizons (e.g., \FIG{model_of_delays_and_horizons}B, blue, green, and orange initial timepoints for the 3-month forecast horizon).

Since submission dates were not available prior to April 2005, our analysis of natural A/H3N2 sequences spanned from April 1, 2005 to October 1, 2019.
To simplify the data required for these analyses, we produced forecasts of natural A/H3N2 populations with our best sequence-only model from our prior work \citep{Huddleston2020}, a composite model based on local branching index (LBI) \citep{Neher:2014eu} and mutational load \citep{Luksza:2014hj}.
For simulated A/H3N2-like populations, we produced forecasts with the ``true fitness'' model that relies on the normalized fitness value of each simulated sample.

Each forecast generated a predicted future frequency per sequence in the initial timepoint's tree.
As in our prior work, we calculated the earth mover's distance \citep{Rubner1998} between the predicted and observed future populations using HA amino acid sequences from initial and future timepoints, predicted future frequencies from the initial timepoint, and observed future frequencies from future timepoint.
For the future timepoint, we used data from the ``no delay'' scenario as our truth set, regardless of the delay scenario for the initial timepoint.
This design allowed us to measure the effect of ideal and realistic submission delays on forecast accuracy relative to a scenario with no delays.

\subsection{Defining clades}

Official clade definitions do not exist for all time periods of our analysis of A/H3N2 populations and do not exist at all for simulated A/H3N2-like populations.
Therefore, we defined ``clades'' for natural and simulated HA trees based on the amino acid haplotypes of HA at 49 previously defined epitope sites \citep{Luksza:2014hj}.
This definition of clades follows from realistic clade definitions that tend to depend on mutations in HA1 at known epitope sites.
Although this automated haplotype-based definition of clades tended to create monophyletic groups, the same haplotype could emerge multiple times independently in different phylogenetic clades.

For each tree in our analysis, we assigned clade labels to each tip and internal node based on their amino acid haplotype at epitope sites.
For each tip, this clade label was deterministic across trees, as the sequence of a tip did not change.
Tips also inherited the clade labels of their ancestral nodes in the tree, allowing us to estimate frequencies of larger clades based on the frequencies of tips in those clades.
However, the inferred sequence at internal nodes varied with the tips present in a given tree.
For example, submission delays could change the tips present in trees representing the same timepoint, resulting in different internal nodes and inferred sequences.
Additionally, the continued accumulation of amino acid mutations between timepoints changed the composition of internal nodes over time, complicating the comparison of large clades between initial and final timepoints of a forecast.

To stably define clades for internal nodes in our analysis, we created a single tree with all HA sequences present in our analysis and without delayed sequence submission.
We inferred ancestral sequences for internal nodes in this tree and assigned clades to each node and tip based on their amino acid haplotypes.
We created a mapping between each tip and the clades it descended from that allowed us to track clade frequencies across 6-year trees built per timepoint in the analysis.

\subsection{Estimating current and future clade frequencies}

We estimated current clade frequencies for each timepoint in our analysis by summing the frequencies of tips in a given timepoint's tree by the clade assigned to each tip.
To inspect the effects of submission delays on clade frequency estimates, we calculated the clade frequency error per timepoint and clade by subtracting the clade frequency estimated with ideal or realistic delayed sequence submission from the corresponding clade frequency without delays.
We compared the effects of submission delays for clades of different sizes by filtering clades by their frequency estimated without delays to small clades ($\ge$1\% and $<$10\%) and large clades ($\ge$10\%).

We estimated future clade frequencies for each combination of delay type, initial timepoint, and final timepoint in the analysis using the following steps.
First, we merged the frequencies per tip at each timepoint with all forecasts from the same timepoint.
The resulting table included the initial and predicted frequencies for each tip for each combination of initial timepoint, forecast horizon, and submission delay.
To define the observed future population for each forecast, we selected all tips that were present in the final timepoint of the corresponding dataset without submission delays.
Using the tip-to-clade mapping described above, we found all clades that were present at both initial and final timepoints.
We calculated the initial and predicted future frequencies per clade at the initial timepoint by summing the frequencies of all tips in each clade.
We calculated the observed future frequencies per clade at the final timepoint with the same approach.
Finally, we merged the initial and predicted future frequencies with the observed future frequencies by clade, keeping all clades that were present in the initial timepoint.
To focus our analysis on the most biologically relevant clades, we limited our analysis to clades that had an initial frequency $\ge$10\% and $\le$95\% under the scenario with no delay.

\section{Acknowledgments}

TKTK

\nocite{*} % This command displays all refs in the bib file. PLEASE DELETE IT BEFORE YOU SUBMIT YOUR MANUSCRIPT!
\bibliography{delays}

\end{document}
